src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -244,7 +244,7 @@
           short-summary: The ID of a subnet in an existing VNet into which to deploy the cluster.
         - name: --workspace-resource-id
           type: string
-          short-summary: The resource ID of an existing Log Analytics Workspace to use for storing monitoring data.
+          short-summary: If not specified, uses default Log Analytics workspace if exists else creates one. Specify resource ID of an existing Log Analytics Workspace to use for storing monitoring data.
```

Comment:
Minor wording changes:
`The resource ID of an existing Log Analytics Workspace to use for storing monitoring data. If not specified, use the default Log Analytics Workspace if it exists, otherwise create one.`

Reply:
Fixed the wording. I kept 'uses' instead of 'use' since the CLI command is going to figure the default workspace automatically not the user . Please let me know if the updated wording not clear we can discuss.